Lib/shutil.py

```diff
@@ -136,6 +136,7 @@ def _fastcopy_sendfile(fsrc, fdst):
     # changes while being copied.
     try:
         blocksize = max(os.fstat(infd).st_size, 2 ** 23)  # min 8MB
+        blocksize = min(blocksize, 2 ** 31 - 2)  # max 2GB
     except Exception:
```

Comment:
What exceptions besides `OSError` can be raised? I do not feel comfortable with silencing a wide range of exceptions.

Reply:
Agreed.